router-tests/cache_warmup_test.go

```diff
@@ -428,6 +451,24 @@ func TestCacheWarmup(t *testing.T) {
 					Query: `query { employees { id details { forename surname } } }`,
 				})
 				require.Equal(t, `{"data":{"employees":[{"id":1,"details":{"forename":"Jens","surname":"Neuse"}},{"id":2,"details":{"forename":"Dustin","surname":"Deus"}},{"id":3,"details":{"forename":"Stefan","surname":"Avram"}},{"id":4,"details":{"forename":"Björn","surname":"Schwenzer"}},{"id":5,"details":{"forename":"Sergiy","surname":"Petrunin"}},{"id":7,"details":{"forename":"Suvij","surname":"Surya"}},{"id":8,"details":{"forename":"Nithin","surname":"Kumar"}},{"id":10,"details":{"forename":"Eelco","surname":"Wiersma"}},{"id":11,"details":{"forename":"Alexandra","surname":"Neuse"}},{"id":12,"details":{"forename":"David","surname":"Stutt"}}]}}`, res.Body)
+
+				// For the next 2 queries below we will:
+				// - miss normalization cache
+				// - hit validation cache
+				// - hit plan cache
+
+				res = xEnv.MakeGraphQLRequestOK(testenv.GraphQLRequest{
```

Comment:
can you duplicate this test with a persisted operation, sending just the sha as the request with different variables, similar to here?